### PR TITLE
[APPS-3459] Update en.yml file with marketplace listing content

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite, @zendesk/i18n 

### Description

Update en.yml file with marketplace listing content. Even though this app is hidden on the marketplace we should still make this update to ensure the installation instructions are still present on the manage page.

### References
* JIRA: https://zendesk.atlassian.net/browse/APPS-3459

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

File: translations/en.yml [hunk 1]
```diff
@@ -6,13 +6,74 @@ packages:
...
+
+[Sign up for a Box account](https://www.box.com) to get started and install the application today.
+
+
+Box® provides a secure, scalable content sharing and collaboration platform that both users and IT love and adopt, including 92% of the Fortune 500. Box's dynamic, flexible content management solution empowers users to share and access content from anywhere, while providing IT enterprise-grade security and oversight into how content moves within their organizations.
+
+
+Legal Disclaimer:
+
+Use of the Attachment Viewer will result in an HTML 5 replica\"\" of your attachment being stored in your Box account.  Use of the Box service is subject to the terms and conditions you agree to when setting up your Box account and constitutes an Other Service as that term is defined in the Zendesk [Terms of Service](http://www.zendesk.com/company/terms)."
```
Comment (injuhwang) [inline translations/en.yml]: Shouldn't the link be https://www.zendesk.com/company/customers-partners/master-subscription-agreement now?

File: translations/en.yml [hunk 2]
```diff
@@ -6,13 +6,74 @@ packages:
...
+4. Enter the following required information, then click Install:
+
+  * Token: Your Box API token.
+
+5.  Refresh your Zendesk Support account and you'll see the Attachment Viewer icon in the sidebar.
+
+
+Legal Disclaimer:  
+
+Use of the Attachment Viewer will result in an HTML 5 \"\"replica\"\" of your attachment being stored in your Box account.  Use of the Box service is subject to the terms and conditions you agree to when setting up your Box account and constitutes an Other Service as that term is defined in the Zendesk [Terms of Service](http://www.zendesk.com/company/terms)."
```
Comment (injuhwang) [inline translations/en.yml]: The URL should be changed to https://www.zendesk.com/company/customers-partners/master-subscription-agreement now.

**Comment by pmgrove** [comment]
@injuhwang 💇

**Comment by injuhwang** [comment]
👍

**Pull request merged**